packages/package_fiji_20141125/.shed.yml

```diff
@@ -1,13 +0,0 @@
-name: package_fiji_20141125
-owner: iuc
-description: Contains a tool dependency definition that downloads and extracts version
-  20141125 of Fiji.
```

Comment:
@gregvonkuster no need to remove the packages. We need to maintain them ... or wait ... they never hit the MTS, right? In this case let's kill them :)